library/aes.c

```diff
@@ -1272,7 +1292,10 @@ int mbedtls_aes_crypt_cfb128( mbedtls_aes_context *ctx,
 
     *iv_off = n;
 
-    return( 0 );
+end:
+    if( ret != 0 )
+        mbedtls_platform_zeroize( out, orig_length );
```

Comment:
Do we really need to zeroize `out`? Nothing has been written to `out` yet.

Reply:
`out` is a pointer to the original `output`.
If, for example, In lines 1270 and 1287 we write to `output`, and by that updating `out`.
If for example, second iteration in the operation fails, we need to clear the whole output buffer. Or at least, what was written.

Reply:
OK

Yes, missed there was a loop here. All good.

Reply:
Perhaps an optimization would be to clear only the number of bytes written, and not hte whole buffer. For example, if a 4GB buffer is given and the failure happens after 32 bytes, I don't see the point in clearing the full 4GB.